Scripts/check_versions.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
```

Comment:
Any reason these cannot go under eng/scripts? That is the convention we use in other places to try and keep the root of the repo a little cleaner.

Reply:
Definitely should be able to.

Reply:
Done.